ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -180,7 +180,15 @@ def _get_l3_interface_cfg(self, l3_interface: dict) -> dict | None:
         if ip_address == "dhcp" and l3_interface.get("set_default_route", False):
             interface["dhcp_client_accept_default_route"] = True
 
+        if l3_interface.get("enable_wan_acls", True):
+            if ip_address != "dhcp":
+                interface["access_group_in"] = self._get_inbound_wan_acl_name(interface_name)
+            interface["access_group_out"] = "WAN_TRANSPORT_OUT"
+
         if self.shared_utils.cv_pathfinder_role:
             interface["flow_tracker"] = {"hardware": "WAN-FLOW-TRACKER"}
 
         return strip_empties_from_dict(interface)
+
+    def _get_inbound_wan_acl_name(self, interface_name):
+        return f"WAN_TRANSPORT_{interface_name}_IN"
```

Comment:
Please change the ACL names to be `ACL-WAN-TRANSPORT-{interface_name.upper()}-IN`, `ACL-WAN-TRANSPORT-OUT` to be consistent with other generated naming.